ste/sender-blockBlob.go

```diff
@@ -142,7 +142,22 @@ func (s *blockBlobSenderBase) Epilogue() {
 		jptm.Log(pipeline.LogDebug, fmt.Sprintf("Conclude Transfer with BlockList %s", blockIDs))
 
 		// commit the blocks.
-		if _, err := s.destBlockBlobURL.CommitBlockList(jptm.Context(), blockIDs, s.headersToApply, s.metadataToApply, azblob.BlobAccessConditions{}); err != nil {
+		doCommit := func() error {
+			_, commitErr := s.destBlockBlobURL.CommitBlockList(jptm.Context(), blockIDs, s.headersToApply, s.metadataToApply, azblob.BlobAccessConditions{})
+			return commitErr
+		}
+		err := doCommit()
+		if strErr, ok := err.(azblob.StorageError); ok && strErr.ServiceCode() == azblob.ServiceCodeInvalidBlockList {
+			// Experimental debugging code, to be shipped in release 10.4.
+			// Wait a bit (actually a lot) and retry.
+			time.Sleep(20 * time.Second)
```

Comment:
Is this a service bug, or an azcopy bug we're trying to work around here?

Reply:
We don't know what's going on. But, after eliminating all the client-side causes I could think of, this code is here in case its server-side.